lib/plugins/digging.js

```diff
@@ -10,7 +13,7 @@ function inject (bot) {
   function dig (block, cb) {
     if (bot.targetDigBlock) bot.stopDigging()
     cb = cb || noop
-    bot.lookAt(block.position.offset(0.5, 0.5, 0.5), false, () => {
+    bot.lookAt(block.position.offset(0.5, 0.5, 0.5), true, () => {
```

Comment:
Doing instant looking might get people kicked in some servers, could you add an option to dig (default to false) to do "slow" looking ?
(You can have default parameters by doing some tricks with cb, there are other examples of that in the code)